tests/test_conditional_steps.py

```diff
@@ -98,9 +98,9 @@ def _build_step_command(files_to_create, exit_code) -> List[str]:
         return ["bash", "-c", ";".join(commands)]
 
     def _write_config_file(self, steps_info) -> None:
-        true_branch_step: str = f"Step(**{str(steps_info.true_branch_step)})" \
+        true_branch_step: str = f"Configuration([Step(**{str(steps_info.true_branch_step)})])" \
```

Comment:
Please also add test for multiple steps inside if_succeeded and if_failed.
I think it should be enough to just fix the current test.

Reply:
As I wrote in the PR description, I plan to do it in a separate change.

> Tests extension will be prepared in a separate PR.

I want to add additional child `Step`s in tests and it will probably lead to a lot of tests infrastructure code change, that's why I want to put it in a separate PR. Nevertheless, if you insist, I can do it in this PR to prove that the code change works properly.